routes/activities/add.js

```diff
@@ -37,6 +39,8 @@ const handleAdd = async (req, res, reader) => {
             break
         }
         break
+      } else if (typeof resultStack === 'undefined') {
```

Comment:
I would just do `else if (!resultStack) { ... ` That will also work if it returns null for some reason.